File: UNIT 4/Python/rata-rata.py
numbers = [None for i in range(10)]
count = 10
sum = 0
average = 0.0

print("\nMasukan 10 bilangan: ")

for i in range(0, 10):
    numbers[i] = int(input("Data ke-" + str(i + 1) + " > "))
    sum += numbers[i]

average = float(sum) / count
print("\nNilai rata-ratanya adalah: ", average, "\n")

File: UNIT 4/C/rata-rata.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    int i;
    int numbers[10];
    int count = 10;
    long sum = 0L;
    float average = 0.0f;

    printf("Masukan 10 bilangan: \n");

    for (i = 0; i < count; i++) {
        printf("Data ke-%d> ", i + 1);
        scanf("%d", &numbers[i]);
        sum += numbers[i];
    }

    average = (float)sum / count;
    printf("\nNilai rata-ratanya adalah: %.2f\n", average);

    system("pause");
    return 0;
}